Client/sdk/game/Common.h

```diff
@@ -1518,22 +1518,32 @@ enum eVehicleComponent
     VEHICLE_COMPONENT_MAX,
 };
 
-enum eVehicleDummies
+namespace eVehicleDummy
 {
-    LIGHT_FRONT_MAIN = 0,
-    LIGHT_REAR_MAIN,
-    LIGHT_FRONT_SECONDARY,
-    LIGHT_REAR_SECONDARY,
-    SEAT_FRONT,
-    SEAT_REAR,
-    EXHAUST,
-    ENGINE,
-    GAS_CAP,
-    TRAILER_ATTACH,
-    HAND_REST,
-    EXHAUST_SECONDARY,
-    WING_AIRTRAIL,
-    VEH_GUN,
+    enum e
+    {
+        LIGHT_FRONT_MAIN = 0,
+        LIGHT_REAR_MAIN,
+        LIGHT_FRONT_SECONDARY,
+        LIGHT_REAR_SECONDARY,
+        SEAT_FRONT,
+        SEAT_REAR,
+        EXHAUST,
+        ENGINE,
+        GAS_CAP,
+        TRAILER_ATTACH,
+        HAND_REST,
+        EXHAUST_SECONDARY,
+        WING_AIRTRAIL,
+        VEH_GUN,
+        VEH_UNKNOWNDUMMY,
+        TOTAL_INTERNAL = 15,
+
+        // custom dummies for MTA, not part of GTA
+        EXHAUST_LEFT = 15,
+        EXHAUST_RIGHT = 16,
+        TOTAL = 17
```

Comment:
```suggestion
        TOTAL_INTERNAL,

        // custom dummies for MTA, not part of GTA
        EXHAUST_LEFT = TOTAL_INTERNAL,
        EXHAUST_RIGHT,
        TOTAL,
```
Does this work?

Reply:
I have added two new functions for vehicle-specific dummies:
```
setVehicleDummyPosition
getVehicleDummyPosition
```
`setVehicleModelDummyPosition` is a model-specific function, so I thought it would be awkward to have vehicle-specific dummies in there. I have also added a new enum for it.